bin/citgm-all.js

```diff
@@ -29,6 +30,14 @@ const yargs = commonArgs(require('yargs'))
     alias: 'J',
     type: 'boolean',
     description: 'Auto detect number of cores to use to run tests in parallel'
+  })
+  .option('withTags', {
+    type: 'string',
```

Comment:
Is there a reason to not use the [array key](https://www.npmjs.com/package/yargs#arraykey) here